helpers.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+func panicIfError(e error) {
+	if e != nil {
+		fmt.Printf("panic:\n%v\n", e)
```

Comment:
I am pretty sure panic will print the error message. No need to print it before panic itself.

Reply:
When running tests I had no error messages or anything readable whatsoever, neither with verbose.
I can remove it if you want, it is not that relevant now